duo_openvpn_as.py

```diff
@@ -204,20 +203,36 @@ def sign(ikey, skey, method, host, uri, date, sig_version, params):
     Return basic authorization header line with a Duo Web API signature.
     """
     canonical = canonicalize(method, host, uri, params, date, sig_version)
-    if isinstance(skey, unicode):
+    if not isinstance(skey, str):
         skey = skey.encode('utf-8')
-    sig = hmac.new(skey, canonical, hashlib.sha1)
+
+    try:
+        sig = hmac.new(skey, canonical, hashlib.sha1)
+
+    except TypeError:
```

Comment:
I think this except block can go away if we use the text_type check above

Reply:
done